src/clj/rems/workflow/dynamic.clj

```diff
@@ -253,7 +286,47 @@
                      :applicantuserid "applicant"
                      :workflow {:type :workflow/dynamic
                                 :handlers ["assistant"]}}
-        requested (apply-command application {:actor "assistant" :decider "deity" :type ::request-decision})
-        decided (apply-command requested {:actor "deity" :decision :approved :type ::decide})]
+        injections {:valid-user? #{"deity"}}
+        requested (apply-command application {:actor "assistant" :decider "deity" :type ::request-decision} injections)
+        decided (apply-command requested {:actor "deity" :decision :approved :type ::decide} injections)]
     (is (= {:decider "deity"} (select-keys requested [:decider :decision])))
     (is (= {:decision :approved} (select-keys decided [:decider :decision])))))
+
+(deftest test-decision-errors
```

Comment:
Group the tests by command instead of by result. All tests for ::request-decision should be together, both the happy cases and the failure cases. The same for ::decide.